package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Simple music streaming service",
   "private": true,
   "scripts": {
-    "build": "webpack -p",
+    "build": "NODE_ENV='production' webpack -p",
```

Comment:
1 - this won't work on windows. To make this cross platform - use https://www.npmjs.com/package/cross-env.
2 - no need for quotes around 'production'

Reply:
Ok, thanks